api/rust/core/src/db/query.rs

```diff
@@ -15,8 +15,8 @@ impl QueryArg {
 }
 
 pub enum QueryType {
-	SELECT,
 	INSERT,
+	SELECT,
```

Comment:
curious, does the order here make a difference? 🤔

Reply:
Only for readability! It confused me when I wrote the `iota` version in Go.

Reply:
https://github.com/suborbital/reactr/blob/f995e1d03589aa46ef99450dff8c7202ac252f00/api/rust/core/src/db/query.rs#L24-L31

🤯 , great fix! haha